tests/playwright-test/playwright.ct-react.spec.ts

```diff
@@ -240,3 +240,33 @@ test.fixme('should work with stray JS import', async ({ runInlineTest }) => {
   expect(result.exitCode).toBe(0);
   expect(result.passed).toBe(1);
 });
+
+test('should work with JSX in variable', async ({ runInlineTest }) => {
+  const result = await runInlineTest({
+    'playwright/index.html': `<script type="module" src="/playwright/index.js"></script>`,
+    'playwright/index.js': `
+      //@no-header
+    `,
+
+    'src/button.jsx': `
+      //@no-header
+      export const Button = () => <button>Button</button>;
+    `,
+
+    'src/button.test.jsx': `
+      //@no-header
+      import { test, expect } from '@playwright/experimental-ct-react';
+      import { Button } from './button';
+
+      const button = <Button></Button>;
+
+      test('pass button', async ({ mount }) => {
+        const component = await mount(button);
+        await expect(component).toHaveText('Button'); 
+      });
+    `,
```

Comment:
Let's add some tests that show the limits. I.e. cases where we can't do the correct thing (short of solving the halting problem). This will be nice to use for reference in docs.

Reply:
I need to start adding Vue / Svelte tests to show the limits. Otherwise, import-wise, React is somewhat limitless.